File: apps/institution-api/Models/VwProgramPublication.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace ProviderApi.Models
{
    public partial class VwProgramPublication
    {
        public int ProgramPublicationId { get; set; }
        public string ProgramAdvertisedName { get; set; }
        public string ProgramPublicationStatus { get; set; }
        public bool? PublishFundingFlag { get; set; }
        public bool? PublishOnWebsiteFlag { get; set; }
        public int? ProgramId { get; set; }
    }
}

File: libs/common/models/vw-program-publication.ts



export class VwProgramPublication {
    public programPublicationId: number = 0;
    public programAdvertisedName: string = '';
    public programPublicationStatus: string = '';
    public publishFundingFlag: boolean;
    public publishOnWebsiteFlag: boolean;
    public programId: number;

    constructor(init? : Partial<VwProgramPublication>) {
        
        Object.assign(this, init);
    }
}

